core/tests/test_utils.py

```diff
@@ -80,6 +81,51 @@ def empty_environ():
         os.environ['HTTP_HOST'], os.environ['SERVER_PORT'])
 
 
+def get_filepath_from_filename(filename, rootdir):
+    """Returns filepath using the filename.
```

Comment:
This needs way more explanation. What's the context? Why not just do os.path.join(rootdir, filename)? Please explain what is going on more, so that developers who have no idea what this function does can understand it based on the docstring.

Reply:
Updated.